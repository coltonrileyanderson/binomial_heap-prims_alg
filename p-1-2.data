7 1 1 ; 7 1 224 ; 0 1 138 ; 7 5 452 ; 6 5 296 ; 
3 6 160 ; 6 5 335 ; 3 6 241 ; 1 7 25 ; 1 4 200 ; 
5 5 359 ; 6 2 58 ; 7 2 365 ; 5 5 227 ; 0 3 225 ; 
8 5 231 ; 6 1 447 ; 2 2 305 ; 2 0 378 ; 6 0 279 ; 
3 1 213 ; 7 1 338 ; 0 6 66 ; 7 6 265 ; 0 3 90 ; 
5 4 341 ; 8 3 481 ; 3 1 7 ; 3 9 262 ; 5 1 476 ; 
7 4 424 ; 8 4 107 ; 7 0 307 ; 3 5 457 ; 6 6 214 ; 
2 9 226 ; 8 0 301 ; 9 0 480 ; 0 2 3 ; 3 8 248 ; 
9 4 63 ; 3 1 12 ; 3 8 153 ; 9 7 350 ; 5 3 35 ; 
6 9 433 ; 6 1 275 ; 6 3 65 ; 0 8 256 ; 0 6 227 ; 
4 1 460 ; 8 4 170 ; 8 7 193 ; 8 7 449 ; 1 4 498 ; 
1 7 135 ; 4 4 201 ; 7 0 29 ; 5 6 128 ; 3 5 110 ; 
4 6 453 ; 8 8 79 ; 7 3 449 ; 4 2 18 ; 8 2 446 ; 
7 9 299 ; 2 0 100 ; 2 2 354 ; 0 2 393 ; 6 2 220 ; 
4 7 83 ; 2 7 250 ; 7 6 380 ; 0 6 128 ; 4 4 194 ; 
4 4 115 ; 6 4 322 ; 8 1 444 ; 0 6 398 ; 5 7 89 ; 
2 2 447 ; 9 2 81 ; 8 5 365 ; 4 3 344 ; 9 7 116 ; 
3 2 31 ; 9 7 75 ; 2 1 301 ; 9 2 92 ; 6 3 222 ; 
0 0 359 ; 5 3 252 ; 0 0 331 ; 5 6 399 ; 2 3 326 ; 
7 4 205 ; 4 1 40 ; 7 2 79 ; 9 4 234 ; 2 9 193 ; 

