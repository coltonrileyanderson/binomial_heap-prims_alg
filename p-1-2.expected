0: 7
1: 1(7)1 8(7)193
2: 0(1)138 4(8)107 5(8)231
3: 2(4)18 9(4)63
4: 3(2)31 6(2)58
weight: 840
